msu/utils/entities.nut

```diff
@@ -0,0 +1,32 @@
+::MSU.Entities <- {
+	EntityTypeToDefaultFactionMap = {},
+	HighestIDDiff = -99,
+	HighestID = -99,
```

Comment:
why -99? might as well just make it -1

Reply:
`-1` is the ID of `::Const.EntityType.Player`. And I don't want to do `-2`. So I went with an obvious negative number that shows intent by going `-99`. Open to other suggestions.

Reply:
>-1 is the ID of ::Const.EntityType.Player
oh my lol

Reply:
wow pain